### PR TITLE
Gsagh-446

### PR DESCRIPTION
**Pull request by DominikaLos**

### CODE REVIEW

File: GsaGH/Components/5_Results/GetResult.cs [hunk 1]
```diff
@@ -25,8 +25,6 @@ public class GetResult : GH_OasysComponent {
     private ReadOnlyDictionary<int, AnalysisCaseResult> _analysisCaseResults;
     private ReadOnlyDictionary<int, CombinationCaseResult> _combinationCaseResults;
     private Guid _modelGuid;
-    private Dictionary<Tuple<CaseType, int>, GsaResult> _result;
-    private int _tempNodeId;
```
Comment (kpne) [inline GsaGH/Components/5_Results/GetResult.cs:29]: This will remove the caching - not what we want. Why is this needed?

File: GsaGH/Components/4_Analysis/AnalyseModel.cs [hunk 2]
```diff
@@ -256,23 +257,23 @@ string message
...
             }
 
-            string report = model.Model.AnalysisTaskReport(task.Key);
-            string[] split = report.Split('\n');
-            report = split[0].Replace(": ", "\n") + "\n" + string.Join("\n", split.Skip(1));
-            report = report.Replace("Checking", "\nChecks:");
-            report = report.Replace("Solving", "\nSolver:");
-            report += "\n \n";
-            reports.Add(report);
+            string rep = report.Notes[task.Key];
```
Comment (tlmnrnhrdt) [inline GsaGH/Components/4_Analysis/AnalyseModel.cs:260]: why are we shortening variable names?

Comment (tlmnrnhrdt) [reply GsaGH/Components/4_Analysis/AnalyseModel.cs:260]: also this broke something..

you can´t access the notes with the task.Key! all current notes belong to the current task

**Review by kpne — APPROVED** [review]
